src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -80,6 +80,7 @@ namespace System.Runtime.CompilerServices
     public sealed partial class IDispatchConstantAttribute : System.Runtime.CompilerServices.CustomConstantAttribute
     {
         public IDispatchConstantAttribute() { }
+        [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
```

Comment:
This is the only member (besides ctor) of this class. Should the whole class be marked `SupportedOSPlatformAttribute("windows")`?

Reply:
Could be